src/applications/gi/components/vet-tec/VetTecContactInformation.jsx

```diff
@@ -3,62 +3,175 @@ import PropTypes from 'prop-types';
-              <div>{institution.physicalAddress2}</div>
-            )}
-            {institution.physicalAddress3 && (
-              <div>{institution.physicalAddress3}</div>
-            )}
+export const VetTecContactInformation = ({ institution }) => {
+  const firstProgram = institution.programs[0];
+  const primarySCOs = institution.versionedSchoolCertifyingOfficials.filter(
+    SCO => SCO.priority === 'PRIMARY',
+  );
+
+  const secondarySCOs = institution.versionedSchoolCertifyingOfficials.filter(
+    SCO => SCO.priority === 'SECONDARY',
+  );
+
+  /* Production flag for 19871 */
+  if (environment.isProduction()) {
+    return (
+      <div className="additional-information vads-l-grid-container--full">
+        <div className="vads-l-row">
+          <div className="vads-l-col--12 medium-screen:vads-l-col--6">
+            <h3>Physical address</h3>
             <div>
-              {institution.physicalCity}, {institution.physicalState}{' '}
-              {institution.physicalZip}
+              {institution.physicalAddress1 && (
+                <div>{institution.physicalAddress1}</div>
+              )}
+              {institution.physicalAddress2 && (
+                <div>{institution.physicalAddress2}</div>
+              )}
+              {institution.physicalAddress3 && (
+                <div>{institution.physicalAddress3}</div>
+              )}
+              <div>
+                {institution.physicalCity}, {institution.physicalState}{' '}
+                {institution.physicalZip}
+              </div>
             </div>
           </div>
-        </div>
-        <div className="vads-l-col--12 medium-screen:vads-l-col--6">
-          <h3>Mailing address</h3>
-          <div>
-            {institution.address1 && <div>{institution.address1}</div>}
-            {institution.address2 && <div>{institution.address2}</div>}
-            {institution.address3 && <div>{institution.address3}</div>}
+          <div className="vads-l-col--12 medium-screen:vads-l-col--6">
+            <h3>Mailing address</h3>
             <div>
-              {institution.city}, {institution.state} {institution.zip}
+              {institution.address1 && <div>{institution.address1}</div>}
+              {institution.address2 && <div>{institution.address2}</div>}
+              {institution.address3 && <div>{institution.address3}</div>}
+              <div>
+                {institution.city}, {institution.state} {institution.zip}
+              </div>
             </div>
           </div>
         </div>
       </div>
-    </div>
-
-    {/* Production flag for 19534 */}
-    {!environment.isProduction() &&
-      institution.versionedSchoolCertifyingOfficials &&
-      institution.versionedSchoolCertifyingOfficials.length > 0 && (
-        <div>
-          <div className="vads-u-margin-top--4">
-            <h3>School certifying officials</h3>
+    );
+  }
+  return (
+    <div>
+      {institution.physicalAddress1 && (
```

Comment:
can you break from this line 58 to line 137 out into a function (let's say `renderContactDetails`) that calls a function for each piece of information (i.e. `physicalAddress`, `mailingAddress`, `emailAddress`, `phoneNumber`)?